apps/meteor/client/hooks/useIsSelfHosted.ts

```diff
@@ -0,0 +1,9 @@
+import { useStatistics } from '../views/hooks/useStatistics';
+
+export const useIsSelfHosted = (): boolean => {
+	const { data } = useStatistics();
+
+	const isSelfHosted = data?.deploy?.platform !== 'rocket-cloud';
```

Comment:
I think it is important to handle the loading case, otherwise we'll have false/positive cases